### PR TITLE
[DO NOT MERGE] Remove build:ci yarn command from rps and wallet packages

### PR DESCRIPTION
**Pull request by kerzhner**
Fixes https://github.com/statechannels/monorepo/issues/794.

### CODE REVIEW

File: packages/rps/package.json [hunk 1]
```diff
@@ -23,7 +22,7 @@
     "eslint:write": "eslint \"**/*.{js,ts,tsx}\" --fix --max-warnings=0",
     "start": "yarn contracts:compile && node scripts/start.js",
     "start:shared-ganache": "NODE_ENV=development npx start-shared-ganache",
-    "prepare": "yarn build:ci",
+    "prepare": "run-s clearContracts build:typescript contracts:compile",
```
Comment (andrewgordstewart) [inline packages/rps/package.json]: I don't think `clearContracts` is needed.

**Comment by kerzhner** [comment]
@andrewgordstewart do you think we should only have a `build` script defined for the packages that are a dependency in another monorepo package?

**Comment by andrewgordstewart** [comment]
I wasn't quite correct in what I said: some packages should have their _`prepare`_ command removed. The `prepare` command should be run precisely in the packages where preparation is necessary.

File: packages/rps/package.json (hunk 1 above)
Comment (kerzhner) [reply packages/rps/package.json]: Good call.

**Comment by kerzhner** [comment]
Closing in favor of https://github.com/statechannels/monorepo/pull/798

**Pull request closed without merging**